src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -75,7 +75,8 @@ struct EvolutionMetavars {
       SlopeLimiters::Minmod<1, system::variables_tag::tags_list>>;
 
   // public for use by the Charm++ registration code
-  using events = tmpl::list<dg::Events::Registrars::Observe<system>>;
+  using events = tmpl::list<dg::Events::Registrars::Observe<
+      1, system::variables_tag::tags_list, system::variables_tag::tags_list>>;
```

Comment:
I think we should generally use `db::get_variables_tags_list` instead, because that can handle prefixed variables. Not strictly necessary in this case, but could be make someone's life easier.